dash/include/dash/halo/Halo.h

```diff
@@ -68,38 +163,68 @@ class StencilSpec {
 };  // StencilSpec
 
 /**
- * Specifies the values global boundary Halos
+ * Global boundary Halo properties
  */
-enum class Cycle : uint8_t {
+enum class BoundaryProp : uint8_t {
   /// No global boundary Halos
   NONE,
   /// Global boundary Halos with values from the opposite boundary
   CYCLIC,
   /// Global boundary Halos with predefined fixed values
-  FIXED
+  CUSTOM
```

Comment:
I'd prefer `FIXED` instead of `CUSTOM`

Reply:
The reason is, it isn't really fixed, because the user can change it any time.